<?xml version="1.0" encoding="UTF-8"?>
<launch>
        <node pkg="detector" name="real_cam_node" type="real_cam2ros.py" output="screen"> </node>
        <!-- START ARUCO DETECTOR -->
        <node pkg="detector" name="detect_marker" type="aruco_detector_v2.py" output="screen">
                <param name="image_width" value="1280" />
                <param name="image_height" value="720" />
                <param name="image_raw" value="/camera/color/image_raw" />
        </node>
</launch>
